log-collector/src/main/scala/io/phdata/pulse/logcollector/Util.scala

```diff
@@ -0,0 +1,34 @@
+package io.phdata.pulse.logcollector
+
+import io.phdata.pulse.common.domain.LogEvent
+
+object Util {
+
+  /**
+   * Convert a [[LogEvent]] to a flattened [[Map[String,String]]
```

Comment:
You can be specific in this comment and tell you are flattening out the `properties` field only, without that knowledge the user doesn't know what 'flattening' means.